docs/plugins/removal.md

```diff
@@ -0,0 +1,66 @@
+# Removing a Plugin
+
+!!! warning
+    The instructions below detail the general process for removing a NetBox plugin. However, each plugin is different and may require additional tasks or modifications to the steps below. Always consult the documentation for a specific plugin **before** attempting to remove it.
+
+## Disable the Plugin
+
+Disable the plugin by removing it from the `PLUGINS` list in `configuration.py`.
+
+## Remove its Configuration
+
+Delete hte plugin's entry (if any) in the `PLUGINS_CONFIG` dictionary in `configuration.py`.
+
+!!! tip
+    If there's a chance you may reinstall the plugin, consider commenting out any configuration parameters instead of deleting them.
+
+## Re-index Search Entries
+
+Run the `reindex` management command to reindex the global search engine. This will remove any stale entries pertaining to objects provided by the plugin.
```

Comment:
do we want to show this like we do for collecstatic, and should have the venv activated:

python3 manage.py reindex